Fastlane/deployment_lanes.rb

```diff
@@ -280,10 +280,10 @@
       }
     )
 
-    # Delete 1 pre-release found before the release we just created.
-    # This is temporarily set to 1 to test out. We can eventually increase this number slowly
+    # Delete 10 pre-releases found before the release we just created.
+    # This is temporarily set to 10 to test out. We can eventually increase this number slowly
```

Comment:
Is it still temporary?

Reply:
Yes! Eventually, I want to upper it to 20 or even 30 🙂